src/styles/theme/button.ts

```diff
@@ -16,75 +16,209 @@ const baseStyles = {
   '&[data-is-loading]': { cursor: 'wait' },
 };
 
+const primary = {
+  px: 3,
+  py: '6/4',
+  color: 'primary50',
+  bg: 'primary600',
+  borderColor: 'primary600',
+  boxShadow: 'sm',
+
+  lineHeight: '130%',
+  '&:hover': {
+    color: 'primary50',
+    bg: 'primary700',
+    borderColor: 'primary700',
+  },
+
+  ...baseStyles,
+  '&:active': {
```

Comment:
the active state should be a box shadow or something, because changing the border size make the button move